target/linux/ramips/image/mt7621.mk

```diff
@@ -656,26 +656,51 @@ define Device/wsr-600
 endef
 TARGET_DEVICES += wsr-600
 
-define Device/zbt-we1326
-  DTS := ZBT-WE1326
+define Device/zbt-we1326-256m
+  DTS := ZBT-WE1326-256M
   IMAGE_SIZE := $(ralink_default_fw_size_16M)
   DEVICE_VENDOR := ZBT
   DEVICE_MODEL := ZBT-WE1326
+  DEVICE_VARIANT := 256MB RAM
```

Comment:
I think other devices where just using something like "256M" here. I'm not actually requesting a change here, but maybe have a second look ...